app/javascript/components/patient/PatientPage.js

```diff
@@ -10,7 +10,7 @@ class PatientPage extends React.Component {
   render() {
     return (
       <Card id="patient-page" className="mx-2 my-4 card-square">
-        <Card.Header className="h5" id="patient-info-header">
+        <Card.Header as="h1" className="patient-card-header" id="patient-info-header">
```

Comment:
using `className` only styles the element. Styling is useless to a screen reader. We care about semantics. In this case, using `as` is the correct way to set the Header value